src/test/kotlin/pt/up/fe/ni/website/backend/utils/documentation/payloadschemas/model/PayloadActivity.kt

```diff
@@ -89,6 +89,13 @@ class PayloadActivity {
                     "Description of the event",
                     JsonFieldType.STRING,
                     optional = true
+                ),
+                DocumentedJSONField(
+                    "gallery[]",
+                    "Array of photos associated with the activity",
+                    JsonFieldType.ARRAY,
+                    isInRequest = false,
```

Comment:
Given how the image of each `Activity` is being documented thus far, I'd leave leave this line out and specify that what is being saved is the path to these images